bundles/org.openhab.binding.shelly/src/main/resources/ESH-INF/thing/sensor.xml

```diff
@@ -0,0 +1,240 @@
+       <label>Sensors</label>
+       <description>Data from the Sense sensors</description>
+        <channels>
+            <channel id="temperature" typeId="sensorTemp" />
+            <channel id="tempUnit"    typeId="sensorTempUnit" />
+            <channel id="humidity"    typeId="sensorHumidity" />
+            <channel id="lux"         typeId="sensorLux" />
+            <channel id="motion"      typeId="sensorMotion" />
+            <channel id="charger"     typeId="sensorCharger" />
+            <channel id="last_update" typeId="lastUpdate" />
+            <channel id="event"       typeId="eventTrigger" />
+        </channels>
+    </channel-group-type>
+
+   <channel-group-type id="batteryStatus">
+       <label>Battery Status</label>
+       <description>Status of the device's battery</description>
+        <channels>
+            <channel id="batteryLevel" typeId="batLevel" />
+            <channel id="lowBattery"   typeId="batAlert" />
+            <channel id="voltage"      typeId="batVoltage" />
+        </channels>
+    </channel-group-type>
+   <channel-group-type id="shortBatteryStatus">
+       <label>Battery Status</label>
+       <description>Status of the device's battery</description>
+        <channels>
+            <channel id="batteryLevel" typeId="batLevel" />
+            <channel id="lowBattery"   typeId="batAlert" />
+        </channels>
+    </channel-group-type>
+
+   <channel-group-type id="senseControl">
+       <label>Sense Control</label>
+       <description>Sense Settings</description>
+        <channels>
+            <channel id="key" typeId="senseKey" />
+        </channels>
+    </channel-group-type>
+
+    <channel-type id="sensorTemp">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Temperature from the sensor</description>
+        <state readOnly="true" pattern="%.1f" >
+        </state>
+    </channel-type>
+    <channel-type id="sensorTempUnit">
+        <item-type>String</item-type>
+        <label>Temp Unit (C/F)</label>
+        <description>Temperature from the sensor C=Celsius, F=Fahrenheit)</description>
+        <state readOnly="true">
+            <options>
+                <option value="C">Celsius</option>
+                <option value="F">Fahrehheit</option>
+            </options>
+        </state>
+    </channel-type>
+    <channel-type id="sensorHumidity">
+        <item-type>Number</item-type>
+        <label>Humidity</label>
+        <description>Relative humidity in % (0..100%)</description>
+        <state readOnly="true" pattern="%.2f%%" >
+        </state>
+    </channel-type>
+    <channel-type id="sensorFlood">
+        <item-type>Switch</item-type>
+        <label>Flood Alarm</label>
+        <description>TRUE: Indicates flood condition / water detected</description>
+        <state readOnly="true">
+        </state>
+    </channel-type>
+    <channel-type id="sensorRainMode">
+        <item-type>Switch</item-type>
```

Comment:
Device config ?

Reply:
In general yes, but configured by the Shelly App. It flips the meaning of an alarm so the rule etc. should be able to detect the type of alarm. Again: It doesn't make sense to have a device config via App and set it also on the OH side

Reply:
In rainMode, there will only be 2 alarms, water detected ande water not present. rainMode=off will send alarms every 5 Minutes and activate sound alarm when triggered. What difference would a roule detect ? IMHO, this is a onetime config at initial setup.